snuba/datasets/transactions_processor.py

```diff
@@ -106,6 +106,15 @@ def process_message(self, message, metadata=None) -> Optional[ProcessedMessage]:
         processed["environment"] = promoted_tags.get("environment")
 
         contexts = _as_dict_safe(data.get('contexts', None))
+        print(data)
+        if "geo" not in contexts and isinstance(data.get("geo"), dict):
```

Comment:
How come it's pulled from contexts in transactions but from User in events?

Reply:
It is not pulled from context here. We add it to context from data. The check is just to avoid names conflicts.